api/src/controllers/post/mine.js

```diff
@@ -0,0 +1,68 @@
+const mongoose = require('mongoose');
+
+const postUtils = require('../../utils/post-utils');
+
+/**
+ * Create a new Mine record.
+ *
+ * @param {*} args
+ * @param {*} res
+ * @param {*} next
+ * @param {*} incomingObj Mine record to create
+ * @returns new Mine record
+ */
+exports.createRecord = async function (args, res, next, incomingObj) {
+  // TODO: Populate if needed.
+  const flavourFunctions = {};
+  return await postUtils.createRecordWithFlavours(args, res, next, incomingObj, this.createMaster, flavourFunctions);
+};
+
+/**
+ * Performs all operations necessary to create a master Mine record.
+ *
+ * @param {*} args
+ * @param {*} res
+ * @param {*} next
+ * @param {*} incomingObj Mine record to create
+ * @param {*} flavourIds array of flavour record _ids
+ * @returns created master Mine record
+ */
+exports.createMaster = function (args, res, next, incomingObj, flavourIds) {
+  let Mine = mongoose.model('MineBCMI');
+  let mine = new Mine();
+
+  mine._schemaName = 'MineBCMI';
```

Comment:
For master records, the model/schema is `Mine`.

The flavour record will be `MineBCMI`

Reply:
Updated. I wasn't 100% sure about the whole flavour thing when writing this.